### ISSUE TITLE
0.0.8 regression test failed

### ISSUE THREAD

**Issue by chenrui333** [badkeys/badkeys#21]
👋 trying to build the latest release, but run into some regression test failure. 

Looks like the key check behavior got changed, this is the diff output

```
$ badkeys rsa-nprime.key
- rsainvalid/prime_n vulnerability, rsa[2048], rsa-nprime.key
+ 
```

full build log, https://github.com/Homebrew/homebrew-core/actions/runs/8859878864/job/24330140677
relates to Homebrew/homebrew-core#170217

**Comment by dtrodrigues** [comment]
This check was intentionally removed in the last release: https://github.com/badkeys/badkeys/releases/tag/v0.0.8
> Major speedup release:
>  The rsainvalid module contained a check whether the N in an RSA key was prime, which would indicate a defect key. Via profiling with cProfile/snakeviz, I discovered that this check is very computationally expensive. Removing this feature, which is not very valuable, makes badkeys 10x faster.

**Comment by hannob** [comment]
This is expected. See:
https://github.com/badkeys/badkeys/issues/16
https://github.com/badkeys/badkeys/commit/c11b5a03b2f61b692bf58385e12acd44c041e752

The "rsainvalid" module contained a check for RSA keys whether the n value is a prime.
My idea with that module was that I wanted to do some cheap checks for things that would indicate a somehow broken key. Now, the problem with this particular check was that it's not cheap at all - it turned out it slowed down badkeys massively, while not really bringing any value (the only situation you'd ever see such a key if someone deliberately created it - which I did for this test - but such a key would simply be broken and cannot be used). So I removed it - and we got a speedup by a factor of 10.
